typings/lxml/html/__init__.pyi

```diff
@@ -0,0 +1,8 @@
+from __future__ import annotations
+
+from ._element import (
+    HtmlElement as HtmlElement,
```

Comment:
Why use the `import/as` syntax here if you're not changing the name?

Reply:
It's commonly done that way in type-stubs where the file is just "republishing" an import from somewhere else. It avoids an "unused import" error, which I suppose is why the original author does that and definitely why I keep it that way.

The other way I know of to avoid the unused-import error is to use `__all__ = ["HtmlElement"]` for example, but this is more compact and a straight copy-paste from the stub packages these come from.